querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Dispatch, IStoreState } from 'redux/store/types';
+import { currentEnvironmentSelector } from 'redux/environment/selector';
+import { FullHeight } from 'ui/FullHeight/FullHeight';
+import { Table } from 'ui/Table/Table';
+import { getScheduledDocs } from 'redux/scheduledDataDoc/action';
+import { TaskStatusIcon } from 'components/Task/TaskStatusIcon';
+import { NextRun } from './NextRun';
+import { DataDocScheduleActionButtons } from './DataDocScheduleActionButtons';
+import { DataDocName } from './DataDocName';
+import { HumanReadableCronSchedule } from './HumanReadableCronSchedule';
+import { LastRecord } from './LastRecord';
+import { LastRun } from './LastRun';
+import { setCollapsed } from 'redux/querybookUI/action';
+
+import { get } from 'lodash';
+
+import './DataDocScheduleList.scss';
+
+const DataDocScheduleList: React.FC = () => {
+    const dispatch: Dispatch = useDispatch();
+    const environment = useSelector(currentEnvironmentSelector);
+    const dataDocs = useSelector((state: IStoreState) => state.scheduledDocs);
+    const collapsed: boolean = useSelector(
+        (state: IStoreState) => state.querybookUI.isEnvCollapsed
+    );
+
+    useEffect(() => {
```

Comment:
hm, i think its fine to not hide the sidebar, we have something similar called adhoc query where the sidebar is not collasped. Its up to you to make the decision but we will optimize the UI for this kind of view in future

Reply:
Yes, if you want to change something in the future, do it, I don't mind.